rust/rust.bzl

```diff
@@ -366,7 +367,7 @@ def _rust_binary_impl(ctx):
   depinfo = _setup_deps(ctx.attr.deps,
                         ctx.label.name,
                         output_dir,
-                        is_library=False)
+                        allow_cc_deps=False)
```

Comment:
Looks like you forgot to actually allow rust_binary rules to depend on cc_library